src/expandCollapseUtilities.js

```diff
@@ -602,8 +602,8 @@ return {
   },
   findNewEnd: function(node) {
     var current = node;
-    
-    while( !current.inside() ) {
+
+    while(current.data("parent") === node.data("id")) {
```

Comment:
This line seems weird to me. Since _current_ is equal to _node_ before while loop, while loop cannot be functional. It always compares a component with its parent, so never enters the while loop.

Reply:
oh, I see, makes sense